.circleci/config.yml

```diff
@@ -115,6 +115,8 @@ jobs:
   update_staging_sha_gke:
     docker:
       - image: rainforestapp/helm-deploy:latest
+        environment:
```

Comment:
Could/should we update the `helm-deploy` image to include that by default?

Reply:
Probably a good idea, the only thing is that it's a public image so it's nice if it's optional.